apps/wallet/src/ui/app/components/nft-display/NftImage.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva } from 'class-variance-authority';
+
+import Icon, { SuiIcons } from '../icon';
+
+import type { VariantProps } from 'class-variance-authority';
+
+const containerStyles = cva('overflow-hidden', {
+    variants: {
+        animateHover: {
+            true: [
+                'ease-ease-out-cubic duration-400',
+                'group-hover:shadow-blurXl group-hover:shadow-steel/50',
```

Comment:
These being group aware is a little weird, probably not blocking but seems slightly wrong.